### PR TITLE
Followup edits to PR 1873

### PR DESCRIPTION
**Pull request by bfallonf**
#1873

@adellape @tnguyen-rh @ahardin-rh @tpoitras Any issues?

### CODE REVIEW

File: admin_guide/router.adoc [hunk 1]
```diff
@@ -84,13 +84,12 @@ the location to a router pod using two environment variables.
...
-default if no port is specified.
+|The IP address of the syslog server. Port *514* is the default if no port is
+specified.
 
 |`*ROUTER_LOG_LEVEL*`
-|Optional, set to change the log level of HAProxy. If  not set  the default log
-level is `warning`; this can be changed to any of the 8 log levels that
-HAProxy supports.
+|Optional. Set to change the HAProxy log level. If not set, the default log
+level is `warning`. This can be changed to any log level that HAProxy supports.
```
Comment (ahardin-rh) [inline admin_guide/router.adoc]: perhaps change to:

    *warning*

File: admin_guide/router.adoc [hunk 2]
```diff
@@ -100,15 +99,13 @@ $ oc env dc/router ROUTER_SYSLOG_ADDRESS=<dest_ip:dest_port>  ROUTER_LOG_LEVEL=<
 ----
 ====
 
-For example:
+For example, the following sets HAProxy to send logs to 127.0.0.1 with the
+default port *514* and changes the log level to `debug`.
```
Comment (ahardin-rh) [inline admin_guide/router.adoc]: Same here. Should debug be in asterisks since it is a setting, not a command?

**Comment by ahardin-rh** [comment]
@bfallonf just a few comments frome me, which you can take or leave. I know @adellape and I have had some discussion regarding the need to revisit the [doc guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) and clarify usage.

**Comment by bfallonf** [comment]
[revhistory]
|xref:../admin_guide/router.adoc#[Monitoring Routers]
|Added table for Router Syslog Variables.
%

**Comment by bfallonf** [comment]
@ahardin-rh Thanks. I've made the changes as per suggestion.

@adellape Did you end up getting a answer on the initial commit on when this should be released? Wondering if this should be a Future Release of Next Release.

**Comment by adellape** [comment]
@bfallonf I just checked and the dev PR (https://github.com/openshift/origin/pull/8332) is only in the 3.3 code, so I've relabeled the original PR and yours here to enterprise-3.3. Thanks for following up.

**Comment by bfallonf** [comment]
Big thanks, @adellape . I'll merge away.

**Comment by adellape** [comment]
It looks like I fat-fingered the label on *both* PRs?? Changed them from dedicated-3.3 to enterprise-3.3. :confused:

**Pull request merged**